oracle-e2e/deploy.js

```diff
@@ -2,7 +2,7 @@ const path = require('path')
 const shell = require('shelljs')
 
 const envsDir = path.join(__dirname, 'envs')
-const deployContractsDir = path.join(__dirname, 'submodules/poa-bridge-contracts/deploy')
+const deployContractsDir = path.join(__dirname, '../contracts-2.2.0/deploy')
```

Comment:
does it make sense to introduce a module with the constant to keep the name of the directory containing the bridge contracts as so this module and this constant will be used in other files: `oracle-e2e/scripts/deployERC20.js`, `oracle-e2e/test/ercToErc.js`, `oracle-e2e/test/ercToNative.js` and `oracle-e2e/test/nativeToErc.js`. So, later when the directory with contracts is renamed it will be necessary to change one file instead of five files.

Reply:
Done.